_posts/2020/2020-12-08-1465.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: rafaga
+date: 2020-12-08 
+rid: 1465
+rafagas:
+
+- keyw: lakepedia
```

Comment:
I would have used `lakes` since I see the keywords as a way to join other conversations in Twitter but this is very generic so not a big deal.